pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - "examples/*"
```

Comment:
Having all examples in a shared workspace allows resolving all packages at once.